library/spit_results.py

```diff
@@ -34,10 +33,10 @@ def __init__(self, module):
 
     def write_to_csv(self):
         """Output report data to file in csv format"""
-        f_path = self.file_path
+        f_path = os.path.normpath(self.file_path)
         with open(f_path, 'w') as write_file:
             file_size = os.path.getsize(f_path)
-            vals = ast.literal_eval(self.vals)
+            vals = self.vals
```

Comment:
Why are you removing the `ast.literal_eval`? I'm glad to see it gone, but curious why.